### DISCUSSION TITLE
deep_learning/practical_pytorch/pytorch_feedforward_neuralnetwork/

### DISCUSSION THREAD

**Discussion by giscus** [ritchieng/deep-learning-wizard#27] [Website]
# deep_learning/practical_pytorch/pytorch_feedforward_neuralnetwork/

We try to make learning deep learning, deep bayesian learning, and deep reinforcement learning math and code easier. Open-source and used by thousands globally.

https://www.deeplearningwizard.com/deep_learning/practical_pytorch/pytorch_feedforward_neuralnetwork/

**Comment by AblimitAili** [comment]
Here, the linear layer is expressed as y = Ax+b. But the pytorch official documentation shows y = xA(T)+b. So the dot product then addition, (100,784)*(784,1)+(100,1) = (100,1), shouldn't it be  (1,784)*(784,100)+(1,100) = (1,100) ?
